crypto/bls_test.go

```diff
@@ -299,3 +299,171 @@ func TestRemovePubKeys(t *testing.T) {
 		fmt.Sprintf("incorrect key %s, should be %s, keys are %s",
 			partialPk, BLSkey, pks))
 }
+
+// BLS multi-signature
+// batch verification
+//
+// Verify n signatures of the same message under different keys using the fast
+// batch verification technique and compares the result to verifying each signature
+// separately.
+func TestBatchVerify(t *testing.T) {
```

Comment:
could consider splitting the test up into sections, maybe using t.Run blocks
```
t.Run("Should be able to ...", func(t *testing.T) {
...
})
```

will allow you to still keep the single function, but label each block

Reply:
That's a good idea, I will implement this across the library

Reply:
https://github.com/dapperlabs/flow-go/issues/4870